autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,6 +1,135 @@
+import asyncio
+import time
+import datetime
+from typing import List
 from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Course, Instructor, Section, Meeting, Department
+from scraper.models.course import generate_course_id
+from scraper.models.section import generate_meeting_id
+
+def convert_meeting_time(string_time: str):
+    """ Converts a meeting time from a string in format hhmm to datetime.
+            ex) 1245 = 12:45am. 1830 = 6:30 pm."""
+    if string_time == "" or string_time is None:
+        return None
+    hour = int(string_time[0:2])
+    minute = int(string_time[2:4])
+    meeting_time = datetime.time(hour, minute)
+
+    return meeting_time
+
+def parse_meeting_days(meetings_data):
+    """generates a list of seven booleans where each corresponds to a day in the week.
+    The first is Monday and the last is Sunday. If true, there is class that day"""
+    meeting_class_days = [
+        'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+    attend_days = [meetings_data['meetingTime'][day] for day in meeting_class_days]
+
+    return attend_days
```

Comment:
You can just immediately return the list comprehension instead of assigning it to `attend_days` then returning